src/boundaries/stagger_and_step.jl

```diff
@@ -192,17 +192,17 @@ function rand_u!(u, δ, n, stagger_mode, rng)
         a = -log10(δ)
         s = (15-a)*rand(rng) + a
         u .= randn(rng,n)
-        u .= 10.0^-s*u/norm(u)
+        u .*= 10.0^-s/norm(u)
```

Comment:
```suggestion
        u .*= (10.0^(-s))/norm(u)
```

for readability. I can never tell what operation happens first and I don't remember by heart the order of operations.